src/components/views/EditMilestone.jsx

```diff
@@ -229,7 +231,7 @@ class EditMilestone extends Component {
     });
   }
 
-  getEthConversion(date) {
+  getEthConversion = (date) => {
```

Comment:
This is strange to me, can you revert to normal class member functions?

Reply:
Why is it strange? It avoids to bind the class members in the constructors, using the lexical scope.

Reply:
Sure that is clear, but linters don't like it and in general it seems to be discouraged (react, airbnb libs are not accepting code like that it seems)

Reply:
As reported in the [documentation](https://reactjs.org/docs/handling-events.html), this is not discouraged, but I would say is the *old* way oof doing it, as the *fat arrow* is an *ES6* feature.

Anyway, I'll revert the changes cause the linter still complains and maybe we want to discuss it further.

Reply:
I don't consider that an issue: how many times you need to subclass a component? I think in React hemisphere is more about composition than subclassing.
The benchmarks show the ops/s for instantiating 100 times the same component, which in React is not that common, and when it happens the instantiate component is usually stateless, which it means that can be written as a function and have no method binding at all.